radixdlt/src/main/java/org/radix/Radix.java

```diff
@@ -142,6 +143,9 @@ public static void start(RuntimeProperties properties) {
 		// Start mempool receiver
 		globalInjector.getInjector().getInstance(MempoolReceiver.class).start();
 
+		SyncedRadixEngine stateSynchronizer = globalInjector.getInjector().getInstance(SyncedRadixEngine.class);
```

Comment:
I don't really like this, or the mempool receiver handling, but I can't think of anything immediately better.

Reply:
What's the issue?